code/datums/ai/basic_mobs/basic_subtrees/express_happiness.dm

```diff
@@ -0,0 +1,44 @@
+#define HIGH_HAPPINESS_THRESHOLD 0.7
+#define MODERATE_HAPPINESS_THRESHOLD 0.5
+
+/datum/ai_planning_subtree/express_happiness
+	operational_datums = list(/datum/component/happiness)
+	///the key storing our happiness value
+	var/happiness_key = BB_BASIC_HAPPINESS
+	///list of emotions we relay when happy
+	var/static/list/happy_emotions = list(
+		"celebrate happily!",
+		"dances around in excitement!",
+	)
+	///our moderate emotions
+	var/static/list/moderate_emotions = list(
+		"looks satisfied.",
+		"trots around.",
+	)
+	///emotions we display when we are sad
+	var/static/list/depressed_emotions = list(
+		"looks depressed...",
+		"turns its back and sulks...",
+		"looks towards the floor in dissapointment...",
+	)
```

Comment:
ai_planning_subtrees are singletons, right? having these be static is pretty redundant then but it doesn't matter so you don't have to change it if you don't feel like it righ tnow

Reply:
They are singletons but I made it static incase subtypes for this behavior get created. And now that I think about it I should let this subtree use a blackboard key for the emotions list incase people want to override them